pages/homePage/homePage.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
+    <link rel="stylesheet" href="">
+    <link rel="stylesheet" href="./homePage.css">
```

Comment:
you can import your homepage.css into your styles.css file.

Reply:
My design is to drive page-specific css from styles.css, to have more flexible page structure.